admin/controllers/content/views/edit/model.php

```diff
@@ -95,7 +95,7 @@
 			}
 			else {
 				$redirect_to = Config::uripath() . "/admin/content/all/" . $content->content_type;
-				$msg = "Content saved";
+				$msg = 'Content <a href="' . Config::uripath() . '/admin/content/edit/' . $content->id . '/' . $content_type . '">' . $content->title . '</a> ' . ($new_content ? 'created' : 'updated');
```

Comment:
templating?

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09